### PR TITLE
Added two images in the credit.md page

### LINKED ISSUE #1887: Add an image in the **Credit for reproducible research** page
- **Problem:** The Turing Way chapters are better explained with illustrations and images. We have many chapters that are not connected with any illustration yet.

- **Solution:** Below we provide chapters with the details on the image with links to the chapter and image file in The Turing Way. Please edit the chapter file to add images.

- **How-to add images in The Turing Way:** We use the following 'directive' in the chapter file to add different images (text starting and should also end with ``` ). Where <FILE-PATH>) is the location of the image file, <FILE-NAME> is the name of the file, and <IMAGE-DESCRIPTION>) is used to describe the image for screenreading apps.

```
```{figure} <FILE-PATH>
---
name: <FILE-NAME>
alt: <IMAGE-DESCRIPTION>
---
_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. ```
```
``

*You can find more details in [The Turing Way Style Guide](https://the-turing-way.netlify.app/community-handbook/style/style-figures.html#alternative-text)*

**Task**
- Visit the chapter file (link provided)
- Start editing (click on the pencil symbol) 
- Copy-past the 'directive' to add images
- Replace the <FILE-PATH> and <FILE-NAME>
- Provide details for "IMAGE-DESCRIPTION" by visiting the image link provided for each challenge.

- Add an image in the **Credit for reproducible research** page
    * Chapter: "Credit for reproducible research": https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/reproducible-research/credit.md (click the link to directly edit the file - no need to fork) - *This file should be added in the DOI section* 
    * FILE-PATH: ../figures/DOI.jpg
    * FILE-NAME: DOI
    * Provide details for "IMAGE-DESCRIPTION" by visiting the image [here](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/website/figures/DOI.jpg).

### LINKED ISSUE #1888: Add an image in the **Credit for reproducible research** page: ORCID
- **Problem:** The Turing Way chapters are better explained with illustrations and images. We have many chapters that are not connected with any illustration yet.

- **Solution:** Below we provide chapters with the details on the image with links to the chapter and image file in The Turing Way. Please edit the chapter file to add images.

- **How-to add images in The Turing Way:** We use the following 'directive' in the chapter file to add different images (text starting and should also end with ``` ). Where <FILE-PATH>) is the location of the image file, <FILE-NAME> is the name of the file, and <IMAGE-DESCRIPTION>) is used to describe the image for screenreading apps.

```
```{figure} <FILE-PATH>
---
name: <FILE-NAME>
alt: <IMAGE-DESCRIPTION>
---
_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. ```
```
``

*You can find more details in [The Turing Way Style Guide](https://the-turing-way.netlify.app/community-handbook/style/style-figures.html#alternative-text)*

**Task**
- Visit the chapter file (link provided)
- Start editing (click on the pencil symbol) 
- Copy-past the 'directive' to add images
- Replace the <FILE-PATH> and <FILE-NAME>
- Provide details for "IMAGE-DESCRIPTION" by visiting the image link provided for each challenge.

- Add an image in the **Credit for reproducible research** page
    * Chapter: "Credit for reproducible research": https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/reproducible-research/credit.md (click the link to directly edit the file - no need to fork) - *This file should be added in the ORCID section*
    * FILE-PATH: ../figures/OrcidIDs.jpg
    * FILE-NAME: OrcidIDs
    * Provide details for "IMAGE-DESCRIPTION" by visiting the image [here](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/website/figures/OrcidIDs.jpg).

### PR DESCRIPTION
**Pull request by alihumayun**
### Summary

Fixes #1888
Fixes #1887 

* Adding 2 illustration in the credit chapter

### List of changes proposed in this PR (pull-request)

- Adding 2 illustration in the credit chapter

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/reproducible-research/credit.md [hunk 1]
```diff
@@ -106,6 +106,15 @@ You should cite your dataset directly in the paper in places where it is relevan
 You can find examples of these statements in the publishers' (research data) author policies, or below:
 
 ### Data availability statement examples:
+
+```{figure} ../figures/DOI.jpg
+---
+name: DOI
+alt: <IMAGE-DESCRIPTION>
```
Comment (malvikasharan) [inline book/website/reproducible-research/credit.md:113]: Hey @alihumayun can you add description for alt text

File: book/website/reproducible-research/credit.md [hunk 2]
```diff
@@ -106,6 +106,15 @@ You should cite your dataset directly in the paper in places where it is relevan
...
+---
+_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300.
```
Comment (malvikasharan) [inline book/website/reproducible-research/credit.md:115]:
```suggestion
_The Turing Way_ project illustration by Scriberia.
Used under a CC-BY 4.0 licence.
DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807)
```

Comment (malvikasharan) [reply book/website/reproducible-research/credit.md:115]: Adding a license as well.

**Review by malvikasharan — APPROVED** [review]
Thank you @alihumayun for the PR. If you can add the alt text and add the license info in the ORCID image - this PR will be ready to merge.

File: book/website/reproducible-research/credit.md (hunk 1 above)
Comment (alihumayun) [inline book/website/reproducible-research/credit.md:113]:
```suggestion
alt: <This image shows three boxes with materials on top. The main box in the middle has "identifiers" written on it with three discs on top of it that are labelled "data sets". Both boxes by its side have journal articles on top of the. An arrow on the top of the image points to these images as being 'Digital Object Identifiers'. There is text at the bottom of the image which says 'Persistent', 'Unique', 'Trusted'.>
```

File: book/website/reproducible-research/credit.md [hunk 3]
```diff
@@ -138,7 +147,13 @@ Zenodo should automatically be notified and should make a snapshot copy of the c
 - Use the DOI in any citations of your software and tell any collaborators and users to do the same!
 
 ## Making sure YOU get cited by using ORCID to increase your visibility
-
+```{figure} ../figures/OrcidIDs.jpg
+---
+name: OrcidIDs
+alt: <IMAGE-DESCRIPTION>
```
Comment (alihumayun) [inline book/website/reproducible-research/credit.md:153]:
```suggestion
alt: <This image shows a female researcher in the centre with her arms raised. She is surrounded in an orbit by different types of documents. On the top left there is a caption stating 'ORCID IDS', while on the bottom right there is text that says: 'One ring to rule all outputs'.  >
```

File: book/website/reproducible-research/credit.md [hunk 4]
```diff
@@ -138,7 +147,13 @@ Zenodo should automatically be notified and should make a snapshot copy of the c
...
+---
+_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300.
```
Comment (alihumayun) [inline book/website/reproducible-research/credit.md:155]:
```suggestion
_The Turing Way_ project illustration by Scriberia.Used under a CC-BY 4.0 licence.DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807)
```

**Review by alihumayun — COMMENTED** [review]
Added alt text and added a license to second image

**Pull request merged**